File: samples/msteams-application-resourcehub/Source/microsoft-teams-apps-selfhelp/ClientApp/src/models/feedback-type.ts
export enum FeedbackType {
    /**Indicates that status is not Internal */
    GeneralFeedback = 0,

    /** Indicates that status is external.
     */
    LearningContentFeedback = 1,
    FeedbackFromLearningPath = 2,
    FeedbackFromLearningPathCompleted = 3
}

File: samples/msteams-application-resourcehub/Source/Microsoft.Teams.Apps.Selfhelp.Shared/Models/Enum/FeedbackType.cs
namespace Microsoft.Teams.Apps.Selfhelp.Shared.Models.Enum
{
    /// <summary>
    /// Enum represents types of feedback.
    /// </summary>
    public enum FeedbackType
    {
        /// <summary>
        /// This represents  type is general feedback.
        /// </summary>
        GeneralFeedback = 0,

        /// <summary>
        /// This represents the type LearningContentFeedback.
        /// </summary>
        LearningContentFeedback = 1,

        /// <summary>
        /// This represents the type FeedbackFromLearningPath.
        /// </summary>
        FeedbackFromLearningPath = 2,

        /// <summary>
        /// This represents the type FeedbackFromLearningPathCompleted.
        /// </summary>
        FeedbackFromLearningPathCompleted = 3,
    }
}